src/tests/calculator.test.js

```diff
@@ -0,0 +1,35 @@
+import calculate from '../logic/calculate';
+
+describe('Test for  calculate function', () => {
```

Comment:
- [ ] All the functionalities and scenarios of the calculate module should be covered. Let's kindly cover the scenario where buttonName === 'AC ', buttonName === '.', buttonName === '+/-'.